fMain.Designer.cs

```diff
@@ -30,957 +30,891 @@ protected override void Dispose(bool disposing)
+            this.cAdditional.Name = "cAdditional";
+            this.cAdditional.Size = new System.Drawing.Size(98, 17);
+            this.cAdditional.TabIndex = 12;
+            this.cAdditional.Text = "Everything else";
+            this.cAdditional.UseVisualStyleBackColor = false;
+            this.cAdditional.CheckedChanged += new System.EventHandler(this.cAdditional_CheckedChanged);
+            // 
+            // cGossip
+            // 
+            this.cGossip.AutoSize = true;
+            this.cGossip.BackColor = System.Drawing.Color.Transparent;
+            this.cGossip.CheckAlign = System.Drawing.ContentAlignment.MiddleRight;
+            this.cGossip.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.cGossip.ForeColor = System.Drawing.Color.Black;
+            this.cGossip.Location = new System.Drawing.Point(196, 45);
+            this.cGossip.Name = "cGossip";
+            this.cGossip.Size = new System.Drawing.Size(112, 17);
+            this.cGossip.TabIndex = 11;
+            this.cGossip.Text = "Gossip stone hints";
+            this.cGossip.UseVisualStyleBackColor = false;
+            this.cGossip.CheckedChanged += new System.EventHandler(this.cGossip_CheckedChanged);
+            // 
+            // cSoS
+            // 
+            this.cSoS.AutoSize = true;
+            this.cSoS.BackColor = System.Drawing.Color.Transparent;
+            this.cSoS.CheckAlign = System.Drawing.ContentAlignment.MiddleRight;
+            this.cSoS.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.cSoS.ForeColor = System.Drawing.Color.Black;
+            this.cSoS.Location = new System.Drawing.Point(165, 22);
+            this.cSoS.Name = "cSoS";
+            this.cSoS.Size = new System.Drawing.Size(143, 17);
+            this.cSoS.TabIndex = 10;
+            this.cSoS.Text = "Exclude Song of Soaring";
+            this.cSoS.UseVisualStyleBackColor = false;
+            this.cSoS.CheckedChanged += new System.EventHandler(this.cSoS_CheckedChanged);
+            // 
+            // cMixSongs
+            // 
+            this.cMixSongs.AutoSize = true;
+            this.cMixSongs.BackColor = System.Drawing.Color.Transparent;
+            this.cMixSongs.CheckAlign = System.Drawing.ContentAlignment.MiddleRight;
+            this.cMixSongs.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
+            this.cMixSongs.ForeColor = System.Drawing.Color.Black;
+            this.cMixSongs.Location = new System.Drawing.Point(22, 22);
+            this.cMixSongs.Name = "cMixSongs";
+            this.cMixSongs.Size = new System.Drawing.Size(122, 17);
+            this.cMixSongs.TabIndex = 3;
+            this.cMixSongs.Text = "Mix son
```

Comment:
Please don't hard-code the version name into this text. Don't want to have to modify the designer file every time we do a release.

Reply:
I'll try to find a way to add it without hardcoding.